openapi/platform/platform.yaml

```diff
@@ -1466,6 +1466,81 @@ paths:
               properties:
                 data:
                   $ref: '#/components/schemas/salesAccounts'
+  /reviewRequests:
```

Comment:
I think rather than adding this to the platform yaml file, these endpoints should be placed in their own yaml file in somthing like `openapi > customervoice > customervoice.yaml`.  That's the pattern we went with for AdIntel.